src/Diagnostics.DataProviders/AscClient.cs

```diff
@@ -215,6 +231,10 @@ private static T CastTo<T>(object obj)
 
         private async Task<T> GetAscResponse<T>(HttpRequestMessage requestMessage, bool isBlobRequest, CancellationToken cancellationToken = default(CancellationToken))
         {
+            if (!string.IsNullOrWhiteSpace(SubscriptionLocationPlacementId) && SubscriptionLocationPlacementId.Equals("geos_2020-01-01", StringComparison.CurrentCultureIgnoreCase))
```

Comment:
Hawk had a good point in his PR that its prob wise to put this value in the app-setting instead of code just to avoid any sort of  compliance issue